docs/tutorials/example_case_files/a_example_stavanger_with_commands.yaml

```
@@ -0,0 +1,84 @@
+time:
+  starttime: 2024-05-31 22:00:00 # UTC time stamped, for Oslo this means midnight June 1st.
+  endtime: 2024-06-02 22:00:00 # Typically set to a later time than 24 hrs such that the optimization accounts for next day too.
+  timeunit: minute
+  timeresolution:
+    2024-05-31 22:00:00: 60 # Time resolution in minutes
+
+model:
+  reservoir:
+    Dale:
+      vol_head: # Volume as a function of head
+        ref: 0 # Not used in this case
+        x: # Volume in million m^3
+          - 0
+          - 10
+        y: # Meters above sea level
+          - 900
+          - 1000
+      max_vol: 10 # million m^3
+      lrl: 900 # Lowest regulated Level
+      hrl: 1000 # Highest regulated Level
+      start_head: 950 # Initial head/water level
+      energy_value_input: 24 # Local energy value of the reservoir, values between sale/buy price are sensible
+
+      inflow: # Water flowing to the reservoir as a time series with units m^3/s
+        2024-05-31 22:00:00: 22
+        2024-06-01 12:00:00: 12
+        2024-06-01 18:00:00: 0
+
+  plant:
+    Sola:
+      main_loss: # Friction loss as water flows through tunnel
+        - 0 # Liste av ett element
+      penstock_loss: # Water might be spit into different penstocks per turbines.
+        - 0
+      outlet_line: 30 # Meters above sea level where water leaves turbine. Needed for water pressure calculations.
+
+  generator:
+    Sola_G1:
+      penstock: 1
+      p_max: 10000 # max production i MW -- Here just set to be very high
+      turb_eff_curves: # A function of discharge through the turbine
+        - ref: 950 # Drop height
+          x: # Flow (m^3/s)
+            - 3
+            - 10
+          y: # Efficiency as a percentage
+            - 0.9
+            - 0.95
```

```suggestion
            - 90
            - 95
```
Jeg tok feil her - det skal være tall i prosent